scripts/pestle/helpers/createFile.js

```diff
@@ -27,6 +28,20 @@ function createFile(filePath, template = "", overwriteFile = false) {
   }
 }
 
+// Helper function to append lines.
+function appendToFile(filePath, template = "") {
+  try {
+    fs.writeFileSync(filePath, template, {
+      encoding: "utf8",
+      flag: "a",
+    });
+    console.log("modified:", filePath);
```

Comment:
Do you want to keep this `console.log()`?

Reply:
🤔 there's one for adding a file, so maybe one for modifications too? 
https://github.com/acl-services/paprika/blob/b556073d222b9ef4d8dd5fc5200284980e78d38b/scripts/pestle/helpers/createFile.js#L22